Packs/MicrosoftGraphMail/Integrations/MicrosoftGraphMail/MicrosoftGraphMail.py

```diff
@@ -1150,14 +1152,45 @@ def delete_mail_command(client: MsGraphClient, args):
     return_outputs(human_readable, entry_context)
 
 
+def item_result_creator(raw_response, user_id):
+    item = raw_response.get('item', {})
+    item_type = item.get('@odata.type', '')
+    if 'message' in item_type:
+        message_id = raw_response.get('id')
+        item['id'] = message_id
+        mail_context = build_mail_object(item, user_id=user_id, get_body=True)
+        entry_context = {'MSGraphMail(val.ID === obj.ID)': mail_context}
+        human_readable = tableToMarkdown(
+            f'Attachment ID {message_id} \n **message details:**',
+            mail_context,
+            headers=['ID', 'Subject', 'SendTime', 'Sender', 'From', 'HasAttachments', 'Body']
+        )
+        return_outputs(
+            human_readable,
+            entry_context,
+            raw_response=raw_response
+        )
+    else:
+        human_readable = f'Integration does not support  attachments from type {item_type}'
+        return_outputs(human_readable, raw_response=raw_response)
+
+
+def create_attachment(raw_response, user_id):
+    attachment_type = raw_response.get('@odata.type', '')
+    if 'itemAttachment' in attachment_type:
+        return item_result_creator(raw_response, user_id)
+    if 'fileAttachment' in attachment_type:
+        return demisto.results(file_result_creator(raw_response))
+    return {}
```

Comment:
shouldn't this raise an error? as the customer run the `get_attachment_command` and he is expecting to get a file